packages/core/src/meta/MetaManagerImpl.ts

```diff
@@ -57,15 +60,15 @@ export default class MetaManagerImpl extends MetaManager {
   /**
    * @inheritdoc
    */
-  setMetaName(name: string, value: string) {
-    this._metaName.set(name, value);
+  setMetaName(name: string, value: MetaValue, otherAttrs?: MetaAttributes) {
```

Comment:
can we rename `value` to `content`?  So it matches the actual values.

(don't forget on the `MetaManager` class)